docs/source/api_ref_models.rst

```diff
@@ -184,21 +187,19 @@ To download the Phi-3 Mini 4k instruct model:
     phi3.lora_phi3_mini
     phi3.qlora_phi3_mini
     phi3.phi3_mini_tokenizer
-    phi3.Phi3MiniTokenizer
-
 
 mistral
 -------
 
 All models from `Mistral AI family <https://mistral.ai/technology/#models>`_.
 
-Request Access on `Hugging Face <https://huggingface.co/mistralai/Mistral-7B-v0.3>`__.
+Important: You need to request access on `Hugging Face <https://huggingface.co/mistralai/Mistral-7B-v0.1>`__ to download this model.
```

Comment:
Where did we land on this? Why not just use 0.2 if the arch is the same?

Reply:
i can run it and see if loss goes down, but thats not the most robust test

Reply:
Eh np, fine to just keep it as is for now